### PR TITLE
Fix tuple arguments type in function

### LINKED ISSUE #1348: BUG: [LLVM backend] Tuple as function args
```py

def foo(x: tuple[i32, i32], y: tuple[i32, i32]) -> bool:
    return x[0] == y[0]

def f():
    a11: tuple[i32, i32]
    b11: tuple[i32, i32]
    a11 = (1, 2)
    b11 = (1, 2)
    print(foo(a11, b11))

f()
```

Error:
```
code generation error: asr_to_llvm: module failed verification. Error:
Stored value type does not match pointer operand type!
  store i32 1, i32** %0, align 4
 i32*Stored value type does not match pointer operand type!
  store i32 2, i32** %1, align 4
 i32*Stored value type does not match pointer operand type!
  store i32 1, i32** %8, align 4
 i32*Stored value type does not match pointer operand type!
  store i32 2, i32** %9, align 4
 i32*
```

### PR DESCRIPTION
**Pull request by Smit-create**
Fixes #1348

### CODE REVIEW

File: src/libasr/codegen/asr_to_llvm.cpp [hunk 1]
```diff
@@ -2984,7 +2984,8 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
                     ASR::storage_typeType local_m_storage = ASR::storage_typeType::Default;
                     llvm_el_types.push_back(get_arg_type_from_ttype_t(asr_tuple->m_type[i], m_abi,
                                             arg_m_abi, local_m_storage, arg_m_value_attr, local_n_dims,
-                                            local_a_kind, is_local_array_type, ASRUtils::intent_local));
+                                            local_a_kind, is_local_array_type,
+                                            ASRUtils::intent_local)->getPointerElementType());
```
Comment (czgdp1807) [inline src/libasr/codegen/asr_to_llvm.cpp:2988]: You can't use `getPointerElementType` (not allowed from LLVM 15 onwards). So please figure out a different way to achieve the same goal that you are trying to by using this API.

**Review by czgdp1807 — CHANGES_REQUESTED** [review]

File: src/libasr/codegen/asr_to_llvm.cpp [hunk 2]
```diff
@@ -2977,15 +2977,15 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
...
                 for( size_t i = 0; i < asr_tuple->n_type; i++ ) {
-                    bool is_local_array_type = false;
-                    int local_n_dims = 0;
-                    int local_a_kind = -1;
-                    ASR::storage_typeType local_m_storage = ASR::storage_typeType::Default;
-                    llvm_el_types.push_back(get_arg_type_from_ttype_t(asr_tuple->m_type[i], m_abi,
-                                            arg_m_abi, local_m_storage, arg_m_value_attr, local_n_dims,
-                                            local_a_kind, is_local_array_type,
-                                            ASRUtils::intent_local)->getPointerElementType());
+                    llvm_el_types.push_back(get_type_from_ttype_t(asr_tuple->m_type[i],
```
Comment (czgdp1807) [inline src/libasr/codegen/asr_to_llvm.cpp:2986]: You can use `get_type_from_ttype_t_util` as well. It won't require you to declare auxiliary argument variables.

**Comment by certik** [comment]
What is the status of this? @czgdp1807 can you please review?

**Comment by czgdp1807** [comment]
Already reviewed. See https://github.com/lcompilers/lpython/pull/1349#discussion_r1038949491. Waiting for @Smit-create to update.

File: src/libasr/codegen/asr_to_llvm.cpp (hunk 2 above)
Comment (Smit-create) [reply src/libasr/codegen/asr_to_llvm.cpp]: Thanks! Updated

**Review by certik — APPROVED** [review]
Looks great, a simple change.

**Pull request merged**